test/RunTests

```diff
@@ -669,6 +669,7 @@ sub run_tests() {
                 else {
                     my $errfile = valgrind_errfile($TestNo);
                     warn "$0: test $TestNo: FAILED: valgrind errors in $errfile\n";
+                    next;
```

Comment:
What does the next do?

Reply:
It continues running the other tests instead of stopping here